config.js

```diff
@@ -381,6 +381,11 @@ const schema = {
       format: Number,
       default: 1800,
       env: 'AUTH_TOKEN_TTL'
+    },
+    privateKey: {
+      doc: 'Private key for signing JSON Web Tokens',
+      format: String,
+      default: 'YOU-MUST-CHANGE-ME-NOW!'
```

Comment:
@eduardoboucas any thoughts on failing to boot if this hasn't been changed?

Reply:
I did wonder about that and I'm torn. On the one hand, it'd make sense to throw an error if you fail to change that value, as that's an important security flaw. On the other hand, it might confuse people if a fresh installation of the app simply fails to boot.

Perhaps we could show a warning if the key hasn't been changed. Also, if we get around to building something that performs a production-ready checklist on the various apps, like we discussed the other way (somewhere), this could be one of the things it'd check for.

But I'm happy to change the approach if you think we should remove the default and make the app fail.

Reply:
Not, you're right, it's a similar issue for the other products - trying to strike a balance between fast bootstrap and solid security.